docs/getting-started/security-ui.asciidoc

```diff
@@ -32,6 +32,20 @@ The navigation menu contains direct links and expandable groups, identified by t
 [role="screenshot"]
 image::images/nav-overview.gif[Overview of the navigation menu]
 
+[[visualization-actions]]
+[float]
+== Visualization actions
+
+Many {elastic-sec} histograms, graphs, and tables display an *Inspect* button (image:images/inspect-icon.png[Inspect icon,19,19]) when you hover over them. Click to examine the {es} queries used to retrieve data throughout the app. 
+
+[role="screenshot"]
+image::images/inspect-icon-context.png[Inspect icon,40%]
+
+Other visualizations display an options menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]), which allows you to inspect the visualization's queries, add it to a new or existing case, or open it in Lens for customization.
```

Comment:
So this is a small UX nit -- or question rather. I thought if the menu title is visible in the UI, that we bold it? In this case, we can see it says `Options`. Reviewing the [UI guidelines](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui), but can't tell for sure. 
```suggestion
Other visualizations display an **Options** menu (image:images/three-dot-icon.png[Three-dot menu icon,18,18]), which allows you to inspect the visualization's queries, add it to a new or existing case, or open it in Lens for customization.
```

Reply:
I went with no bold, lowercase here because the menu button doesn't have a label or tooltip. It feels a little misleading to tell the user about a menu called "**Options**" when they won't be able to find it by name until they've already opened it. Referring to it generally as a menu of options doesn't set up the same kind of expectation.

The [emphasis](https://docs.elastic.dev/content-design/emphasis) writing guidelines don't quite cover this. They say to use bold for "menu items" and "UI controls (when the user interacts with the control)" — but in this case **Options** is a menu _title_, not item, and the user never directly clicks or selects anything called "Options." It'd be nice to expand the guidelines to include unnamed UI controls, because we encounter those a lot.

Reply:
@joepeeples makes sense, thanks. I believe the guidelines say somewhere that if it's unnamed, then we don't bold it -- however this one is tricky because it _does_ appear as named in the UI technically, but as you said, you don't see it until you click the icon. This is a nuance we probably haven't thought about but agree it would be nice to define in our guidelines.